### PR TITLE
Cp oc cli from cli image to ipi-deprovision

### PR DESCRIPTION
**Pull request by hongkailiu**

### CODE REVIEW

File: images/ipi-deprovision/Dockerfile [hunk 1]
```diff
@@ -3,6 +3,8 @@ LABEL maintainer="hongkliu@redhat.com"
 
 ARG JQ_VERSION=1.6
 
+COPY --from=cli /usr/bin/oc usr/bin
```
Comment (stevekuznetsov) [inline images/ipi-deprovision/Dockerfile]: I think you'll want the target to be an absolute path

Comment (hongkailiu) [reply images/ipi-deprovision/Dockerfile]: checked the base image (hive) ... the `workdir` is `/`

I can NOT use absolute path here

https://github.com/openshift/release/blob/8be771c27b5bbe59557f7cd92cdf3f7fe67bc7f9/ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml#L97

See https://github.com/openshift/ci-tools/pull/40

If everything goes through, I will give a try with absolute path here too.

**Comment by hongkailiu** [comment]
/hold
Want to try `Add` command in the Dockerfile

**Comment by hongkailiu** [comment]
/hold cancel

`Add` works too.

**Comment by hongkailiu** [comment]
/hold cancel

https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_ci-tools/42/pull-ci-openshift-ci-tools-master-images/128/artifacts/build-logs/ipi-deprovision.log

```
# podman run --entrypoint "/bin/bash" -it --rm registry.svc.ci.openshift.org/ci-op-081l7sb3/pipeline:ipi-deprovision
[root@b50670073a21 /]# which oc
/usr/bin/oc

```

**Comment by hongkailiu** [comment]
```
# podman run --entrypoint "/bin/bash" -it --rm registry.svc.ci.openshift.org/ci/ipi-deprovision
Trying to pull registry.svc.ci.openshift.org/ci/ipi-deprovision...Getting image source signatures
...
[root@a2cdece6f285 /]# which oc
/usr/bin/oc

```

**Pull request merged**